tests/config_test.py

```diff
@@ -129,3 +137,13 @@ def testConfigOverrideFlag(self):
     config = configs.GetUserConfig()
     self.assertEqual(config['a']['vm_groups']['default']['vm_count'], 5)
     self.assertEqual(config['a']['flags']['flag'], 'value')
+
+  def testConfigFlagStillParsed(self):
+    flags.DEFINE_list('list_flag', ['default_val'], 'List flag.')
```

Comment:
This will probably linger beyond the lifetime of the test. I suggest creating a separate FlagValues object and passing it in as the flag_values arg of DEFINE_list.